lib/morpheus.gi

```diff
@@ -660,6 +660,8 @@ end);
 ##  dom   a set of elements on which automorphisms act faithful
 ##  aut   Subgroup of already known automorphisms
 ##  condition function that must return `true' on the homomorphism.
+##  setrun  If set to true approximate a run through sets by having the
+##          class indices increasing.
 ##
```

Comment:
My concern is that it's never used anywhere in the library, so if the code at some point gets broken, we'll have no way of knowing. But, we should have a better way of testing things like this (I personally just use .g files which, when everything goes fine, don't print anything out).